specification.md

```diff
@@ -957,21 +1005,20 @@ String? getMessage()
 Returns the message associated with this status, if any.
 
 
-<a name="jobstatus-isterminal"></a>
+<a name="jobstatus-isfinal"></a>
 ```java
-boolean isTerminal()
+boolean isFinal()
 ```
 
 A convenience wrapper for
-[`status.getState().isTerminal()`](#jobstate-isterminal).
+[`status.getState().isFinal()`](#jobstate-isFinal).
 
 
 
 ### JobState
 
 An enumeration holding the possible job states, which are: `NEW`,
-`QUEUED`, `ACTIVE`, `SUSPENDED`, `RESUMED`, `COMPLETED`, `FAILED`,
-`CANCELLED`.
+`QUEUED`, `ACTIVE`, `COMPLETED`, `FAILED`, and `CANCELLED`.
```

Comment:
If we intentionally dropped `SUSPENDED` and `RESUMED`, they should be dropped from the state diagram too.

The state diagram also still has `DONE` rather than `COMPLETED`.

Reply:
Ack

Reply:
done